src/js/components/Video/Video.js

```diff
@@ -413,6 +441,33 @@ const Video = forwardRef(
   },
 );
 
+Video.propTypes = {
+  messages: PropTypes.shape({
+    scrubber: PropTypes.string,
+    meter: PropTypes.string,
+    openMenu: PropTypes.string,
+    closeMenu: PropTypes.string,
+    playButton: PropTypes.string,
+    pauseButton: PropTypes.string,
+    volumeUp: PropTypes.string,
+    volumeDown: PropTypes.string,
+    fullScreen: PropTypes.string,
+  }),
+};
+Video.defaultProps = {
+  messages: {
```

Comment:
The messages will be translated to `aria-label` which usually uses a lower case syntax.